### ISSUE TITLE
HI, would you explan ” the relevant user-item interactions are not included in the validation/test sets to avoid data leakage“ please?

### ISSUE THREAD

**Issue by IIM-D** [HKUDS/RLMRec#20]
HI, would you explan ” the relevant user-item interactions are not included in the validation/test sets to avoid data leakage“ please?
Aren't the test set and validation set split from the binary adjacency matrix of user-item interactions based on a certain proportion?
look forward to your reply!
Best regards!

**Comment by Re-bin** [comment]
Hi 👋!

Thanks for your interest in RLMRec! I think your question is about the response to [this issue](https://github.com/HKUDS/RLMRec/issues/10).

It means that, the review information (which is assigned to user-item interactions) that we use for profile generation is **only taken from the training dataset**. Interactions in the validation and test datasets are not included when generating user and item profiles.

I hope the above answer is helpful to you! 😊

Best regards,
Xubin

**Comment by IIM-D** [comment]
Thanks! Have a nice life and good luck with your studies!
